neurom/core/types.py

```diff
@@ -47,11 +47,12 @@ class NeuriteIter(OrderedEnum):
 @unique
 class NeuriteType(OrderedEnum):
     '''Enum representing valid tree types'''
-    undefined = 1
-    soma = 2
-    axon = 3
-    basal_dendrite = 4
-    apical_dendrite = 5
+    undefined = 0
```

Comment:
@mgeplf I know we just discussed keeping the value different but I just did it for practicality reason as section_type is now accessed by a dict

Reply:
I don't follow why it being access by a dict makes a difference?